manpages/pihole-FTL.conf.5

```diff
@@ -0,0 +1,74 @@
+.TH "pihole-FTL.conf" "5" "pihole-FTL.conf" "pihole-FTL.conf" "June 2018"
+.SH "NAME"
+
+pihole-FTL.conf - FTL's config file
+.br
+.SH "DESCRIPTION"
+
+/etc/pihole/pihole-FTL.conf will be read by \fBpihole-FTL(8)\fR on startup.
+.br
+
+SOCKET_LISTENING=localonly|all
```

Comment:
The options should line up with those in the documentation: https://docs.pi-hole.net/ftldns/configfile/